pkgs/development/cuda-modules/cudnn/fixup.nix

```diff
@@ -40,10 +40,16 @@ finalAttrs: prevAttrs: {
 
   # Tell autoPatchelf about runtime dependencies.
   # NOTE: Versions from CUDNN releases have four components.
-  postFixup = strings.optionalString (strings.versionAtLeast finalAttrs.version "8.0.5.0") ''
-    ${meta.getExe' patchelf "patchelf"} $lib/lib/libcudnn.so --add-needed libcudnn_cnn_infer.so
-    ${meta.getExe' patchelf "patchelf"} $lib/lib/libcudnn_ops_infer.so --add-needed libcublas.so --add-needed libcublasLt.so
-  '';
+  postFixup =
+    strings.optionalString
+      (
+        strings.versionAtLeast finalAttrs.version "8.0.5.0"
+        && strings.versionOlder finalAttrs.version "9.0.0.0"
+      )
+      ''
+        ${meta.getExe' patchelf "patchelf"} $lib/lib/libcudnn.so --add-needed libcudnn_cnn_infer.so
```

Comment:
Since nixfmt inflates the diff anyway, `getExe' patchelf "patchelf"` is just `getExe patchelf`

Reply:
Fixed.